src/env.h

```diff
@@ -1048,6 +1049,9 @@ class Environment final : public MemoryRetainer {
 
   inline void RemoveHeapSnapshotNearHeapLimitCallback(size_t heap_limit);
 
+  inline v8::CpuProfilingResult StartCpuProfile(v8::Local<v8::String> title);
+  inline v8::CpuProfile* StopCpuProfile(v8::Local<v8::String> title);
```

Comment:
In the spirit of #43712, these methods could be non-inline just fine

Reply:
Done !